adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -31,7 +31,7 @@
 from digitalio import Direction
 from adafruit_bus_device.spi_device import SPIDevice
 
-__version__ = "0.0.0-auto.0"
+__version__ = "3.5.3"
```

Comment:
Please undo this. The original version is automatically replaced.

Reply:
no problem undoing that.  FYI:  pre-commit added it.